cloud/azure/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "app_service_default_hostname" {
+  value = "https://${azurerm_app_service.civiform_app.default_site_hostname}"
+}
```

Comment:
EOF newline here too

Reply:
I always miss these!

Reply:
Easy to do! I just have a little script in emacs that ensures it's there whenever I save a file.

Reply:
Easy to miss* I mean